### DISCUSSION TITLE
No completion in go case statements without curly brackets

### DISCUSSION THREAD

**Discussion by AsafMah** [community/community#77167] [Copilot Conversations]
### Select Topic Area

Bug

### Body

IDE: Jetbrains IDEA
Language: Go

See comments in the code for the bug

```go
	switch {
	case i:
		var // no copilot completion here (returns no results)
	case j:
		{
		var  // copilot completion here works (note brackets)
		}
	}
```

**Comment by Akash1134** [comment]
Hi @AsafMah ,

Please refer our [troubleshooting guide](https://docs.github.com/en/copilot/troubleshooting-github-copilot/viewing-logs-for-github-copilot-in-your-environment) to diagnose the issue you are facing with Copilot. 

Thanks!
